csrankings-m.csv

```diff
@@ -1493,6 +1494,7 @@ Mengjuan Liu,UESTC,https://www.is.uestc.edu.cn/teachers.do?id=1085,NOSCHOLARPAGE
 Mengmeng Ge,RMIT University,https://www.rmit.edu.au/contact/staff-contacts/academic-staff/g/ge-dr-mengmeng,dj2SBEIAAAAJ
 Mengshu Hou,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/2057.htm,NOSCHOLARPAGE
 Mengwei Xu,BUPT,https://xumengwei.github.io,hFmJwKQAAAAJ
+Meng Yan,Chongqing University,https://yanmeng.github.io,UBTGNMwAAAAJ

```

Comment:
Remove (see comment above).

Reply:
This entry is still here. Needs to be removed.

Reply:
Hi emeryberger, i removed another duplicated one. Is this necessary to stay?

Reply:
All need to be removed. This is a disambiguation entry. Please read the instructions.